tests/test_testdb_cnxn.py

```diff
@@ -0,0 +1,6 @@
+def test_testdb(cnxn, cursor):
+    """tests the test db connection"""
+    assert cnxn
+    cnxn_parameters = cnxn.get_dsn_parameters()
+    assert cnxn_parameters.get("dbname") == "questioner_test"
+    assert cursor
```

Comment:
Same to here

Reply:
`cnxn` and `cursor` have been declared as fixtures in the `conftest.py` file. Once a fixture is declared in that file, it should be able to be reused in all the test files. Making those assertions here just confirms that the `cnxn` and `cursor` are actually available and functional. I can now use them confidently in the other test files without testing them as long as the assertions here are `True`. If the assertions here were to fail, that would mean that either the `cnxn` and `cursor` fixtures are unavailable, or, that something is actually wrong with the connection. I could skip that test and just use them, but double checking that they worked seemed like a good idea since the db connection will be a dependency in many of the other tests. If you feel like the test is unnecessary though, please let me know and I'll remove them.